mirrord-operator/templates/crd.yaml

```diff
@@ -1468,6 +1468,64 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
+  name: mirrordprofiles.profiles.mirrord.metalbear.co
```

Comment:
Note to myself, add `get` and `list` permission to access this resource for operator basic user

Reply:
Added the permission to `mirrord-operator-user` role. Tested it manually with a service account and confirmed it fetched the namespaced profile successfully.